test/typescript/components.spec.tsx

```diff
@@ -53,6 +53,7 @@ import Table, {
 } from '../../src/Table';
 import { withStyles, StyleRulesCallback } from '../../src/styles';
 import { withResponsiveFullScreen, DialogProps } from '../../src/Dialog';
+import { WithStyles } from '../../src/styles/withStyles';
```

Comment:
should we add this to the exports of `material-ui/styles`, so people don't have multiple imports? :)

Reply:
`WithStyles` is already exported in `styles` no? 🤔

Reply:
Yes, sir ... I am going to blame this on Firday morning 😴

Reply:
I don't think `withStyles` is currently exported in `material-ui/styles` since it keeps coming back as undefined for me.

Reply:
@amyasapp these comments were about `WithStyles`, which is a type... but it certainly seems like [`withStyles` is exported from `styles`](https://github.com/callemall/material-ui/blob/f653691cffa6490dd33122f268c8e625e3e753a0/src/styles/index.js#L4).

Reply:
Hmm, I'm using v0.19.4 and not v1 beta. Would that change anything? My coworkers and I grep'd for `withStyles` and couldn't find it anywhere in our `node_modules`.

Reply:
@amyasapp yeah, I don't think there's a `withStyles` in `v0.19.4`.